tests/test_pyfepdf.py

```diff
@@ -0,0 +1,314 @@
+        codigo,
+        ds,
+        qty,
+        umed,
+        precio,
+        bonif,
+        iva_id,
+        imp_iva,
+        importe,
+        despacho,
+        dato_a,
+    )
+
+    # descuento general (a tasa 21%):
+    u_mtx = cod_mtx = codigo = None
+    ds = u"Bonificación/Descuento 10%"
+    qty = precio = bonif = None
+    umed = 99
+    iva_id = 5
+    if tipo_cbte in (1, 2, 3, 4, 5, 34, 39, 51, 52, 53, 54, 60, 64):
+        # discriminar IVA si es clase A / M
+        imp_iva = -2.21
+    else:
+        imp_iva = None
+    importe = -12.10
+    chk2 = fepdf.AgregarDetalleItem(
+        u_mtx,
+        cod_mtx,
+        codigo,
+        ds,
+        qty,
+        umed,
+        precio,
+        bonif,
+        iva_id,
+        imp_iva,
+        importe,
+        "",
+    )
+
+    # descripción (sin importes ni cantidad):
+    u_mtx = cod_mtx = codigo = None
+    qty = precio = bonif = iva_id = imp_iva = importe = None
+    umed = 0
+    ds = u"Descripción Ejemplo"
+    chk3 = fepdf.AgregarDetalleItem(
+        u_mtx,
+        cod_mtx,
+        codigo,
+        ds,
+        qty,
+        umed,
+        precio,
+        bonif,
+        iva_id,
+        imp_iva,
+        importe,
+        "",
+    )
+
+    assert chk1 == True
+    assert chk2 == True
+    assert chk3 == True
+
+
+def test_agregar_iva():
+    iva_id = 5  # 21%
+    base_imp = 100
+    importe = 21
+    chk = fepdf.AgregarIva(iva_id, base_imp, importe)
+    assert chk == True
+
+
+def test_agregar_tributo():
+    tributo_id = 99
+    desc = "Impuesto Municipal Matanza"
+    base_imp = "100.00"
+    alic = "1.00"
+    importe = "1.00"
+    chk = fepdf.AgregarTributo(tributo_id, desc, base_imp, alic, importe)
+
+    assert chk == True
+
+def test_agregar_cmp_asoc():
+    tipo = 5
+    pto_vta = 2
+    nro = 1234
+    chk = fepdf.AgregarCmpAsoc(tipo, pto_vta, nro)
+
+    assert chk == True
+
+def test_crear_plantilla():
+    sys.argv = []
+
+    config = SafeConfigParser()
+    config.read(CONFIG_FILE)
+    conf_fact = dict(config.items("FACTURA")) 
+
+    fepdf.CrearPlantilla(
+        papel=conf_fact.get("papel", "legal"),
+        orientacion=conf_fact.get("orientacion", "portrait"),
+    )
+
+def test_procesar_plantilla():
+    sys.argv = []
+    sys.argv.append('--debug')
+
+    config = SafeConfigParser()
+    config.read(CONFIG_FILE)
+    conf_fact = dict(config.items("FACTURA")) 
+
+    chk = fepdf.ProcesarPlantilla(
+        num_copias=int(conf_fact.get("copias", 1)),
+        lineas_max=int(conf_fact.get("lineas_max", 24)),
+        qty_pos=conf_fact.get("cant_pos") or "izq",
+    )   
+
+    assert chk == False
+
+
+def test_generar_qr():
+    fepdf.CUIT = "30000000007"
+    assert fepdf.GenerarQR()
+
+def test_main_prueba():
+    sys.argv = []
+    sys.argv.append("--prueba")
+    sys.argv.append("--debug")
+    main()
+
+def test_main_cargar():
+    sys.argv = []
+    sys.argv.append("--prueba")
+    sys.argv.append("--cargar")
+    main()
```

Comment:
After calling main, please do a sanity check s
Example:
```
assert pyfpdf.fepdf.factura ==  {...}
```
If there is no check, the test only assures that the code is executed and no serious error is raised (eg NameError)
It cannot warranty that the code works as expected